tests/test_location.py

```diff
@@ -68,18 +68,21 @@ async def test_system_modes(aresponses, auth_success_response):
         "/api/v2/locations/mmnnoopp/systemMode",
         "post",
         aresponses.Response(text=None, status=204),
+        body_pattern='{"target": "away"}',
     )
     aresponses.add(
         "api-gw.meetflo.com",
         "/api/v2/locations/mmnnoopp/systemMode",
         "post",
         aresponses.Response(text=None, status=204),
+        body_pattern='{"target": "home"}',
```

Comment:
Is this the entire body that you expect? If so, the normal way to do it in `aresponses` is the via the `text` kwarg of the `aresponses.Response` class (similar to line 77). That parameter just takes text that represents the response body.